src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/generate.cmd

```diff
@@ -8,7 +8,7 @@ setlocal
 
 set autoRestVersion=0.17.0-Nightly20160824
```

Comment:
@NonStatic2014 AutoRest is now at version 1.0.x. Please refer to aka.ms/autorest/install for more information on how to install and use the latest version(s).